quiz/file_reader.py

```diff
@@ -0,0 +1,51 @@
+"""
+Module handle file operations
+"""
+from quiz.question import Question
+from quiz.key import Key
+
+
+class FileReader:
+    """
+    Class to perform file operations
+    """
+
+    @staticmethod
+    def read_questions(file_path):
+        """
+        Read questions from file and return list of questions
+        """
+        questions = []
+        try:
+            with open(file_path) as file:
+                lines = file.read().splitlines()
+
+                for question_text in lines:
+                    if question_text:
+                        # Read first character as id and from fourth character onward as question text
+                        questions.append(Question(question_text[:1], question_text[3:]))
```

Comment:
what if file is not formatted correctly? we can validate that too and raise exceptions